modules/ROOT/pages/clustering/databases.adoc

```diff
@@ -225,6 +225,109 @@ neo4j@neo4j> DRYRUN REALLOCATE DATABASES;
 +----------------------------------------------------------------------------------------------------------------------------------------+
 ----
 
+[role=label--new-5.24]
+[[recreate-databases]]
+== Recreate databases using a procedure
+
+Neo4j 5.24 introduces the xref:reference/procedures.adoc#procedure_dbms_cluster_recreateDatabase[`dbms.cluster.recreateDatabase()`] procedure, which allows you:
+
+* To restore a database store to a specified backup, while keeping all the users and roles (metadata) for the database.
+
+* To restore a database to a write mode again after it has been lost (for example, due to a disaster).
```

Comment:
On both this and the previous one we have the word "restore". This is unfortunately very overloaded, could we find a nice way of saying it without this word?